features/change-event.yml

```
@@ -0,0 +1,7 @@
+name: Change event
+description: The `change` event is fired for `<input>`, `<select>`, and `<textarea>` elements when the user modifies the element's value. Unlike the input event, the change event is not necessarily fired for each alteration to an element's value.
```

Mostly for consistency with other event descriptions:

```suggestion
description: The `change` event for `<input>`, `<select>`, and `<textarea>` elements fires when the user modifies the element's value. Unlike the `input` event, the change event does not necessarily fire on every alteration to the element's value.
```